### ISSUE TITLE
Auth is failing on Mac desktop app because of Office.context is null

### ISSUE THREAD

**Issue by malsabbagh** [OfficeDev/office-js#1228]
Authentication is failing using officeJS on Outlook Mac. 

## Expected Behavior
Office-js-helpers should be able to send the token to the main client after a redirect call from oauth. 

## Current Behavior
An exception is thrown in OfficeJS code because Office.contenxt is null. The exception message "DialogError: Cannot close dialog"
PC Desktop and other online browser clients don't have this issue. It is specific to Mac Desktop

## Steps to Reproduce, or Live Example

* Link to live example: (I can provide this later but should be fairly easy to reproduce)

* Additional details:
  1. Normal OAuth flow using officeJS. 
  2. After we get the token the popup fails to send the parsed token back. 

## Context

OAuth authentication using OfficeJS mechanism. 

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: Mac Desktop
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: 16.37 (20051002)
* Operating System: Mac OS 10.15.5

## Useful logs

- [x] Console errors
- [x] Screenshots
- [ ] Test file (if only happens on a particular file)
![image](https://user-images.githubusercontent.com/13791232/85144623-6cf2cf00-b219-11ea-8abc-f87590a586d7.png)

![image](https://user-images.githubusercontent.com/13791232/85143369-a6c2d600-b217-11ea-884b-6b97e8f9120f.png)

![image](https://user-images.githubusercontent.com/13791232/85143693-0f11b780-b218-11ea-8712-cc5c75bc618f.png)

**Comment by malsabbagh** [comment]
Hi @casieber @Zlatkovsky, I know office-js-helper is deprecated now but is there a planned alternative for it?

**Comment by Zlatkovsky** [comment]
@malsabbagh, caseiber had left the company some time ago, and I also left the Office Extensibility team earlier this year.  Tagging @LouMM and @sumurthy , who are the current owners (and who were part of the office-js-helpers deprecation decision)

**Comment by lindalu-MSFT** [comment]
Hi, assigning to the @exextoc Outlook add-ins team to triage! Thanks, lindalu

**Comment by exextoc** [comment]
This is already tracked by Bug 1545877<https://o365exchange.visualstudio.com/DefaultCollection/O365%20Core/_workitems/edit/1545877>: New GitHub Issue [OfficeDev/office-js]: Auth is failing on Mac desktop app because of Office.context is null

**Comment by lindalu-MSFT** [comment]
@exextoc THANK you for the update. 

This is being tracked by internal bug, no date but it is under investigation!

**Comment by exextoc** [comment]
Hey @malsabbagh we are unable to reproduce this on Outlook Mac on the latest build. Could you provide a link to your add-in manifest for us to test, with instructions to reproduce the same?

**Comment by malsabbagh** [comment]
Hey @exextoc, oddly enough I can't reproduce it anymore. We had this issue with a few individuals. But I have followed up with them and it seems the issue is no longer reproducible on their end. One individual reported the issue being cleared after a reboot (not sure if this has anything to do with the issue). OfficeJS code was not modified for any of the individuals who reported the issue. 

The issue initially was reported to us around mid March, 2020. But I was not able to reproduce it at that time and when I tried recently I was able to. It could have been that I was on the wrong version of Mac Desktop. If this is true, maybe the issue was with the following Mac Desktop versions 16.35,16.36 and it was fixed with 16.37 but required a reboot or something. (I am purely speculating here.)

I was using Mac Desktop 16.37 (20051002). The issue first was reproducible. I have done a reboot since, so maybe that fixed it and it is no longer reproducible with 16.37 version. 
I am updating now to 16.38 to validate that it works as expected as well.
